pkg/filematches/filematches.go

```diff
@@ -116,3 +118,11 @@ func FindDraftDeploymentFiles(dest string) (deploymentType string, err error) {
 
 	return "", errors.New("no supported deployment files found")
 }
+
+func GetSubDirPath(dest string, subDir string) (string, error) {
+	cleanPath := strings.TrimRight(dest, "/") + "/" + strings.TrimLeft(subDir, "/")
```

Comment:
instead of using hardcoded path separators, you can use [path.join](https://github.com/Azure/draft/blob/273b704366d6316274de4049283323fb260805dd/pkg/addons/addons.go#L51) which does most of the heavy lifting for you and handles edge cases